pkg/eks/api.go

```diff
@@ -253,16 +239,7 @@ func New(spec *api.ProviderConfig, clusterSpec *api.ClusterConfig) (*ClusterProv
 		provider.ec2 = ec2.New(s, s.Config.Copy().WithEndpoint(endpoint))
 
 	}
-	if endpoint, ok := os.LookupEnv("AWS_ELB_ENDPOINT"); ok {
-		logger.Debug("Setting ELB endpoint to %s", endpoint)
-		provider.elb = elb.New(s, s.Config.Copy().WithEndpoint(endpoint))
 
-	}
-	if endpoint, ok := os.LookupEnv("AWS_ELBV2_ENDPOINT"); ok {
-		logger.Debug("Setting ELBV2 endpoint to %s", endpoint)
-		provider.elbv2 = elbv2.New(s, s.Config.Copy().WithEndpoint(endpoint))
-
-	}
```

Comment:
do we not still need this for the v2 clients?

Reply:
We do, it's handled in the SDK initialization for v2.